<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="eiger.td.teradata.com" name="all" tests="23" failures="17" timestamp="5 Oct 2015 17:31:56 GMT" time="81.228" errors="0">
  <testcase name="dynamicSshClient" time="2.009" classname="com.teradata.tempto.examples.ExampleSshClientUsage"/>
  <testcase name="longRunningCommandTimeout" time="2.974" classname="com.teradata.tempto.examples.ExampleSshClientUsage">
    <failure type="java.lang.AssertionError" message="
Expecting:
 &amp;lt;&amp;quot;com.jcraft.jsch.JSchException: Auth fail&amp;quot;&amp;gt;
to contain:
 &amp;lt;&amp;quot;did not finish within given timeout&amp;quot;&amp;gt; ">
      <![CDATA[java.lang.AssertionError: 
Expecting:
 <"com.jcraft.jsch.JSchException: Auth fail">
to contain:
 <"did not finish within given timeout"> 
	at com.teradata.tempto.examples.ExampleSshClientUsage.longRunningCommandTimeout(ExampleSshClientUsage.java:79)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- longRunningCommandTimeout -->
  <testcase name="sshClientUsage" time="2.516" classname="com.teradata.tempto.examples.ExampleSshClientUsage">
    <failure type="java.lang.RuntimeException" message="com.jcraft.jsch.JSchException: Auth fail">
      <![CDATA[java.lang.RuntimeException: com.jcraft.jsch.JSchException: Auth fail
	at com.teradata.tempto.internal.ssh.JSchSshClient.upload(JSchSshClient.java:98)
	at com.teradata.tempto.examples.ExampleSshClientUsage.sshClientUsage(ExampleSshClientUsage.java:52)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
Caused by: com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:512)
	at com.jcraft.jsch.Session.connect(Session.java:183)
	at com.teradata.tempto.internal.ssh.JSchSshClient.getActiveSession(JSchSshClient.java:78)
	at com.teradata.tempto.internal.ssh.JSchSshClient.upload(JSchSshClient.java:88)
	... 26 more
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- sshClientUsage -->
  <testcase name="selectFromImmutableTable" time="0.194" classname="com.teradata.tempto.examples.PostgresqlQueryTest"/>
  <testcase name="selectFromMutableTable" time="0.166" classname="com.teradata.tempto.examples.PostgresqlQueryTest"/>
  <testcase name="createAndDropMutableTable" time="0.588" classname="com.teradata.tempto.examples.SimpleQueryTest">
    <failure type="java.lang.RuntimeException" message="error invoking methods annotated with com.teradata.tempto.AfterTestWithContext">
      <![CDATA[java.lang.RuntimeException: error invoking methods annotated with com.teradata.tempto.AfterTestWithContext
	at com.teradata.tempto.internal.initialization.TestInitializationListener.invokeMethodsAnnotatedWith(TestInitializationListener.java:277)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.runAfterWithContextMethods(TestInitializationListener.java:266)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.onTestFinished(TestInitializationListener.java:244)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.onTestSuccess(TestInitializationListener.java:219)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.invokeMethodsAnnotatedWith(TestInitializationListener.java:274)
	... 25 more
Caused by: org.junit.ComparisonFailure: expected:<[tru]e> but was:<[fals]e>
	at com.teradata.tempto.examples.SimpleQueryTest.afterTest(SimpleQueryTest.java:87)
	... 30 more
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- createAndDropMutableTable -->
  <testcase name="failingTest" time="0.205" classname="com.teradata.tempto.examples.SimpleQueryTest">
    <failure type="java.lang.RuntimeException" message="error invoking methods annotated with com.teradata.tempto.AfterTestWithContext">
      <![CDATA[java.lang.RuntimeException: error invoking methods annotated with com.teradata.tempto.AfterTestWithContext
	at com.teradata.tempto.internal.initialization.TestInitializationListener.invokeMethodsAnnotatedWith(TestInitializationListener.java:277)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.runAfterWithContextMethods(TestInitializationListener.java:266)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.onTestFinished(TestInitializationListener.java:244)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.onTestFailure(TestInitializationListener.java:226)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.invokeMethodsAnnotatedWith(TestInitializationListener.java:274)
	... 25 more
Caused by: org.junit.ComparisonFailure: expected:<[tru]e> but was:<[fals]e>
	at com.teradata.tempto.examples.SimpleQueryTest.afterTest(SimpleQueryTest.java:87)
	... 30 more
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- failingTest -->
  <testcase name="selectAllFromMultipleTables" time="3.776" classname="com.teradata.tempto.examples.SimpleQueryTest">
    <failure type="java.lang.RuntimeException" message="error invoking methods annotated with com.teradata.tempto.AfterTestWithContext">
      <![CDATA[java.lang.RuntimeException: error invoking methods annotated with com.teradata.tempto.AfterTestWithContext
	at com.teradata.tempto.internal.initialization.TestInitializationListener.invokeMethodsAnnotatedWith(TestInitializationListener.java:277)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.runAfterWithContextMethods(TestInitializationListener.java:266)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.onTestFinished(TestInitializationListener.java:244)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.onTestFailure(TestInitializationListener.java:226)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.invokeMethodsAnnotatedWith(TestInitializationListener.java:274)
	... 25 more
Caused by: org.junit.ComparisonFailure: expected:<[tru]e> but was:<[fals]e>
	at com.teradata.tempto.examples.SimpleQueryTest.afterTest(SimpleQueryTest.java:87)
	... 30 more
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- selectAllFromMultipleTables -->
  <testcase name="selectAllFromMultipleTables" time="0.466" classname="com.teradata.tempto.examples.SimpleQueryTest">
    <failure type="java.lang.RuntimeException" message="error invoking methods annotated with com.teradata.tempto.AfterTestWithContext">
      <![CDATA[java.lang.RuntimeException: error invoking methods annotated with com.teradata.tempto.AfterTestWithContext
	at com.teradata.tempto.internal.initialization.TestInitializationListener.invokeMethodsAnnotatedWith(TestInitializationListener.java:277)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.runAfterWithContextMethods(TestInitializationListener.java:266)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.onTestFinished(TestInitializationListener.java:244)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.onTestFailure(TestInitializationListener.java:226)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.invokeMethodsAnnotatedWith(TestInitializationListener.java:274)
	... 25 more
Caused by: org.junit.ComparisonFailure: expected:<[tru]e> but was:<[fals]e>
	at com.teradata.tempto.examples.SimpleQueryTest.afterTest(SimpleQueryTest.java:87)
	... 30 more
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- selectAllFromMultipleTables -->
  <testcase name="selectAllFromNation" time="0.184" classname="com.teradata.tempto.examples.SimpleQueryTest">
    <failure type="java.lang.RuntimeException" message="error invoking methods annotated with com.teradata.tempto.AfterTestWithContext">
      <![CDATA[java.lang.RuntimeException: error invoking methods annotated with com.teradata.tempto.AfterTestWithContext
	at com.teradata.tempto.internal.initialization.TestInitializationListener.invokeMethodsAnnotatedWith(TestInitializationListener.java:277)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.runAfterWithContextMethods(TestInitializationListener.java:266)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.onTestFinished(TestInitializationListener.java:244)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.onTestFailure(TestInitializationListener.java:226)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.invokeMethodsAnnotatedWith(TestInitializationListener.java:274)
	... 25 more
Caused by: org.junit.ComparisonFailure: expected:<[tru]e> but was:<[fals]e>
	at com.teradata.tempto.examples.SimpleQueryTest.afterTest(SimpleQueryTest.java:87)
	... 30 more
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- selectAllFromNation -->
  <testcase name="selectCountFromNation" time="0.181" classname="com.teradata.tempto.examples.SimpleQueryTest">
    <failure type="java.lang.RuntimeException" message="error invoking methods annotated with com.teradata.tempto.AfterTestWithContext">
      <![CDATA[java.lang.RuntimeException: error invoking methods annotated with com.teradata.tempto.AfterTestWithContext
	at com.teradata.tempto.internal.initialization.TestInitializationListener.invokeMethodsAnnotatedWith(TestInitializationListener.java:277)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.runAfterWithContextMethods(TestInitializationListener.java:266)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.onTestFinished(TestInitializationListener.java:244)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.onTestFailure(TestInitializationListener.java:226)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
	at com.teradata.tempto.internal.initialization.TestInitializationListener.invokeMethodsAnnotatedWith(TestInitializationListener.java:274)
	... 25 more
Caused by: org.junit.ComparisonFailure: expected:<[tru]e> but was:<[fals]e>
	at com.teradata.tempto.examples.SimpleQueryTest.afterTest(SimpleQueryTest.java:87)
	... 30 more
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- selectCountFromNation -->
  <testcase name="testDataSource" time="0.132" classname="com.teradata.tempto.examples.TpchDataSourceTest">
    <failure type="org.junit.ComparisonFailure" message="expected:&amp;lt;&amp;quot;/tempto/tpch/sf-1[,]00/REGION&amp;quot;&amp;gt; but was:&amp;lt;&amp;quot;/tempto/tpch/sf-1[_]00/REGION&amp;quot;&amp;gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<"/tempto/tpch/sf-1[,]00/REGION"> but was:<"/tempto/tpch/sf-1[_]00/REGION">
	at com.teradata.tempto.examples.TpchDataSourceTest.testDataSource(TpchDataSourceTest.java:43)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- testDataSource -->
  <testcase name="sql_tests.testcases.nation.allRows.query_1" time="0.802" classname="com.teradata.tempto.allRows">
    <failure type="java.lang.NoClassDefFoundError" message="com/facebook/presto/hive/$internal/com/google/common/primitives/Ints">
      <![CDATA[java.lang.NoClassDefFoundError: com/facebook/presto/hive/$internal/com/google/common/primitives/Ints
	at org.apache.hive.service.cli.Column.<init>(Column.java:149)
	at org.apache.hive.service.cli.ColumnBasedSet.<init>(ColumnBasedSet.java:51)
	at org.apache.hive.service.cli.RowSetFactory.create(RowSetFactory.java:37)
	at org.apache.hive.jdbc.HiveQueryResultSet.next(HiveQueryResultSet.java:333)
	at com.teradata.tempto.query.QueryResult$QueryResultBuilder.addRows(QueryResult.java:230)
	at com.teradata.tempto.query.QueryResult.forResultSet(QueryResult.java:178)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQueryNoParams(JdbcQueryExecutor.java:102)
	at com.teradata.tempto.query.JdbcQueryExecutor.execute(JdbcQueryExecutor.java:85)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQuery(JdbcQueryExecutor.java:62)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.runTestQuery(SqlQueryConventionBasedTest.java:116)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.test(SqlQueryConventionBasedTest.java:94)
	at com.teradata.tempto.internal.convention.ConventionBasedTestProxyGenerator$ConventionBasedTestProxy.test(ConventionBasedTestProxyGenerator.java:120)
	at com.teradata.tempto.allRows.query_1(Unknown Source)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
Caused by: java.lang.ClassNotFoundException: com.facebook.presto.hive.$internal.com.google.common.primitives.Ints
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 38 more
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- sql_tests.testcases.nation.allRows.query_1 -->
  <testcase name="sql_tests.testcases.nation.allRows.query_2" time="20.011" classname="com.teradata.tempto.allRows">
    <failure type="java.lang.NoClassDefFoundError" message="com/facebook/presto/hive/$internal/com/google/common/primitives/Ints">
      <![CDATA[java.lang.NoClassDefFoundError: com/facebook/presto/hive/$internal/com/google/common/primitives/Ints
	at org.apache.hive.service.cli.Column.<init>(Column.java:149)
	at org.apache.hive.service.cli.ColumnBasedSet.<init>(ColumnBasedSet.java:51)
	at org.apache.hive.service.cli.RowSetFactory.create(RowSetFactory.java:37)
	at org.apache.hive.jdbc.HiveQueryResultSet.next(HiveQueryResultSet.java:333)
	at com.teradata.tempto.query.QueryResult$QueryResultBuilder.addRows(QueryResult.java:230)
	at com.teradata.tempto.query.QueryResult.forResultSet(QueryResult.java:178)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQueryNoParams(JdbcQueryExecutor.java:102)
	at com.teradata.tempto.query.JdbcQueryExecutor.execute(JdbcQueryExecutor.java:85)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQuery(JdbcQueryExecutor.java:62)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.runTestQuery(SqlQueryConventionBasedTest.java:116)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.test(SqlQueryConventionBasedTest.java:94)
	at com.teradata.tempto.internal.convention.ConventionBasedTestProxyGenerator$ConventionBasedTestProxy.test(ConventionBasedTestProxyGenerator.java:120)
	at com.teradata.tempto.allRows.query_2(Unknown Source)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- sql_tests.testcases.nation.allRows.query_2 -->
  <testcase name="sql_tests.testcases.generated.nation.generated_nation" time="0.325" classname="com.teradata.tempto.nation">
    <failure type="java.lang.NoClassDefFoundError" message="com/facebook/presto/hive/$internal/com/google/common/primitives/Ints">
      <![CDATA[java.lang.NoClassDefFoundError: com/facebook/presto/hive/$internal/com/google/common/primitives/Ints
	at org.apache.hive.service.cli.Column.<init>(Column.java:149)
	at org.apache.hive.service.cli.ColumnBasedSet.<init>(ColumnBasedSet.java:51)
	at org.apache.hive.service.cli.RowSetFactory.create(RowSetFactory.java:37)
	at org.apache.hive.jdbc.HiveQueryResultSet.next(HiveQueryResultSet.java:333)
	at com.teradata.tempto.query.QueryResult$QueryResultBuilder.addRows(QueryResult.java:230)
	at com.teradata.tempto.query.QueryResult.forResultSet(QueryResult.java:178)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQueryNoParams(JdbcQueryExecutor.java:102)
	at com.teradata.tempto.query.JdbcQueryExecutor.execute(JdbcQueryExecutor.java:85)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQuery(JdbcQueryExecutor.java:62)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.runTestQuery(SqlQueryConventionBasedTest.java:116)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.test(SqlQueryConventionBasedTest.java:94)
	at com.teradata.tempto.internal.convention.ConventionBasedTestProxyGenerator$ConventionBasedTestProxy.test(ConventionBasedTestProxyGenerator.java:120)
	at com.teradata.tempto.nation.generated_nation(Unknown Source)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- sql_tests.testcases.generated.nation.generated_nation -->
  <testcase name="sql_tests.testcases.sample_hive_table.allRows" time="0.331" classname="com.teradata.tempto.sample_hive_table">
    <failure type="java.lang.NoClassDefFoundError" message="com/facebook/presto/hive/$internal/com/google/common/primitives/Ints">
      <![CDATA[java.lang.NoClassDefFoundError: com/facebook/presto/hive/$internal/com/google/common/primitives/Ints
	at org.apache.hive.service.cli.Column.<init>(Column.java:149)
	at org.apache.hive.service.cli.ColumnBasedSet.<init>(ColumnBasedSet.java:51)
	at org.apache.hive.service.cli.RowSetFactory.create(RowSetFactory.java:37)
	at org.apache.hive.jdbc.HiveQueryResultSet.next(HiveQueryResultSet.java:333)
	at com.teradata.tempto.query.QueryResult$QueryResultBuilder.addRows(QueryResult.java:230)
	at com.teradata.tempto.query.QueryResult.forResultSet(QueryResult.java:178)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQueryNoParams(JdbcQueryExecutor.java:102)
	at com.teradata.tempto.query.JdbcQueryExecutor.execute(JdbcQueryExecutor.java:85)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQuery(JdbcQueryExecutor.java:62)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.runTestQuery(SqlQueryConventionBasedTest.java:116)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.test(SqlQueryConventionBasedTest.java:94)
	at com.teradata.tempto.internal.convention.ConventionBasedTestProxyGenerator$ConventionBasedTestProxy.test(ConventionBasedTestProxyGenerator.java:120)
	at com.teradata.tempto.sample_hive_table.allRows(Unknown Source)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- sql_tests.testcases.sample_hive_table.allRows -->
  <testcase name="sql_tests.testcases.sample_presto_table.allRows" time="0.165" classname="com.teradata.tempto.sample_presto_table">
    <failure type="com.teradata.tempto.query.QueryExecutionException" message="java.sql.SQLException: Error executing query">
      <![CDATA[com.teradata.tempto.query.QueryExecutionException: java.sql.SQLException: Error executing query
	at com.teradata.tempto.query.JdbcQueryExecutor.execute(JdbcQueryExecutor.java:92)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQuery(JdbcQueryExecutor.java:62)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.runTestQuery(SqlQueryConventionBasedTest.java:116)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.test(SqlQueryConventionBasedTest.java:94)
	at com.teradata.tempto.internal.convention.ConventionBasedTestProxyGenerator$ConventionBasedTestProxy.test(ConventionBasedTestProxyGenerator.java:120)
	at com.teradata.tempto.sample_presto_table.allRows(Unknown Source)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
Caused by: java.sql.SQLException: Error executing query
	at com.facebook.presto.jdbc.PrestoStatement.executeQuery(PrestoStatement.java:54)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQueryNoParams(JdbcQueryExecutor.java:101)
	at com.teradata.tempto.query.JdbcQueryExecutor.execute(JdbcQueryExecutor.java:85)
	... 30 more
Caused by: com.facebook.presto.jdbc.internal.airlift.http.client.RuntimeIOException: Server refused connection: http://localhost:8080/v1/statement
	at com.facebook.presto.jdbc.internal.airlift.http.client.ResponseHandlerUtils.propagate(ResponseHandlerUtils.java:17)
	at com.facebook.presto.jdbc.internal.airlift.http.client.FullJsonResponseHandler.handleException(FullJsonResponseHandler.java:55)
	at com.facebook.presto.jdbc.internal.airlift.http.client.FullJsonResponseHandler.handleException(FullJsonResponseHandler.java:35)
	at com.facebook.presto.jdbc.internal.airlift.http.client.jetty.JettyHttpClient.execute(JettyHttpClient.java:297)
	at com.facebook.presto.jdbc.internal.client.StatementClient.<init>(StatementClient.java:96)
	at com.facebook.presto.jdbc.QueryExecutor.startQuery(QueryExecutor.java:63)
	at com.facebook.presto.jdbc.PrestoConnection.startQuery(PrestoConnection.java:588)
	at com.facebook.presto.jdbc.PrestoStatement.executeQuery(PrestoStatement.java:49)
	... 32 more
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at com.facebook.presto.jdbc.internal.jetty.io.SelectorManager.finishConnect(SelectorManager.java:340)
	at com.facebook.presto.jdbc.internal.jetty.io.SelectorManager$ManagedSelector.processConnect(SelectorManager.java:671)
	at com.facebook.presto.jdbc.internal.jetty.io.SelectorManager$ManagedSelector.processKey(SelectorManager.java:640)
	at com.facebook.presto.jdbc.internal.jetty.io.SelectorManager$ManagedSelector.select(SelectorManager.java:607)
	at com.facebook.presto.jdbc.internal.jetty.io.SelectorManager$ManagedSelector.run(SelectorManager.java:545)
	at com.facebook.presto.jdbc.internal.jetty.util.thread.NonBlockingThread.run(NonBlockingThread.java:52)
	at com.facebook.presto.jdbc.internal.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
	at com.facebook.presto.jdbc.internal.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
	at java.lang.Thread.run(Thread.java:745)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- sql_tests.testcases.sample_presto_table.allRows -->
  <testcase name="sql_tests.testcases.sample_presto_table.psqlOverPrestoAllRows" time="0.121" classname="com.teradata.tempto.sample_presto_table">
    <failure type="com.teradata.tempto.query.QueryExecutionException" message="java.sql.SQLException: Error executing query">
      <![CDATA[com.teradata.tempto.query.QueryExecutionException: java.sql.SQLException: Error executing query
	at com.teradata.tempto.query.JdbcQueryExecutor.execute(JdbcQueryExecutor.java:92)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQuery(JdbcQueryExecutor.java:62)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.runTestQuery(SqlQueryConventionBasedTest.java:116)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.test(SqlQueryConventionBasedTest.java:94)
	at com.teradata.tempto.internal.convention.ConventionBasedTestProxyGenerator$ConventionBasedTestProxy.test(ConventionBasedTestProxyGenerator.java:120)
	at com.teradata.tempto.sample_presto_table.psqlOverPrestoAllRows(Unknown Source)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
Caused by: java.sql.SQLException: Error executing query
	at com.facebook.presto.jdbc.PrestoStatement.executeQuery(PrestoStatement.java:54)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQueryNoParams(JdbcQueryExecutor.java:101)
	at com.teradata.tempto.query.JdbcQueryExecutor.execute(JdbcQueryExecutor.java:85)
	... 30 more
Caused by: com.facebook.presto.jdbc.internal.airlift.http.client.RuntimeIOException: Server refused connection: http://localhost:8080/v1/statement
	at com.facebook.presto.jdbc.internal.airlift.http.client.ResponseHandlerUtils.propagate(ResponseHandlerUtils.java:17)
	at com.facebook.presto.jdbc.internal.airlift.http.client.FullJsonResponseHandler.handleException(FullJsonResponseHandler.java:55)
	at com.facebook.presto.jdbc.internal.airlift.http.client.FullJsonResponseHandler.handleException(FullJsonResponseHandler.java:35)
	at com.facebook.presto.jdbc.internal.airlift.http.client.jetty.JettyHttpClient.execute(JettyHttpClient.java:297)
	at com.facebook.presto.jdbc.internal.client.StatementClient.<init>(StatementClient.java:96)
	at com.facebook.presto.jdbc.QueryExecutor.startQuery(QueryExecutor.java:63)
	at com.facebook.presto.jdbc.PrestoConnection.startQuery(PrestoConnection.java:588)
	at com.facebook.presto.jdbc.PrestoStatement.executeQuery(PrestoStatement.java:49)
	... 32 more
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at com.facebook.presto.jdbc.internal.jetty.io.SelectorManager.finishConnect(SelectorManager.java:340)
	at com.facebook.presto.jdbc.internal.jetty.io.SelectorManager$ManagedSelector.processConnect(SelectorManager.java:671)
	at com.facebook.presto.jdbc.internal.jetty.io.SelectorManager$ManagedSelector.processKey(SelectorManager.java:640)
	at com.facebook.presto.jdbc.internal.jetty.io.SelectorManager$ManagedSelector.select(SelectorManager.java:607)
	at com.facebook.presto.jdbc.internal.jetty.io.SelectorManager$ManagedSelector.run(SelectorManager.java:545)
	at com.facebook.presto.jdbc.internal.jetty.util.thread.NonBlockingThread.run(NonBlockingThread.java:52)
	at com.facebook.presto.jdbc.internal.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
	at com.facebook.presto.jdbc.internal.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
	at java.lang.Thread.run(Thread.java:745)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- sql_tests.testcases.sample_presto_table.psqlOverPrestoAllRows -->
  <testcase name="sql_tests.testcases.sample_psql_table.allRows" time="0.106" classname="com.teradata.tempto.sample_psql_table"/>
  <testcase name="sql_tests.testcases.sample_psql_table.allRowsOnPsql2" time="0.117" classname="com.teradata.tempto.sample_psql_table"/>
  <testcase name="sql_tests.testcases.sample_table_insert.insert_0" time="15.525" classname="com.teradata.tempto.sample_table_insert">
    <failure type="java.lang.NoClassDefFoundError" message="com/facebook/presto/hive/$internal/com/google/common/primitives/Ints">
      <![CDATA[java.lang.NoClassDefFoundError: com/facebook/presto/hive/$internal/com/google/common/primitives/Ints
	at org.apache.hive.service.cli.Column.<init>(Column.java:149)
	at org.apache.hive.service.cli.ColumnBasedSet.<init>(ColumnBasedSet.java:51)
	at org.apache.hive.service.cli.RowSetFactory.create(RowSetFactory.java:37)
	at org.apache.hive.jdbc.HiveQueryResultSet.next(HiveQueryResultSet.java:333)
	at com.teradata.tempto.query.QueryResult$QueryResultBuilder.addRows(QueryResult.java:230)
	at com.teradata.tempto.query.QueryResult.forResultSet(QueryResult.java:178)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQueryNoParams(JdbcQueryExecutor.java:102)
	at com.teradata.tempto.query.JdbcQueryExecutor.execute(JdbcQueryExecutor.java:85)
	at com.teradata.tempto.query.JdbcQueryExecutor.executeQuery(JdbcQueryExecutor.java:62)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.runTestQuery(SqlQueryConventionBasedTest.java:116)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.test(SqlQueryConventionBasedTest.java:94)
	at com.teradata.tempto.internal.convention.ConventionBasedTestProxyGenerator$ConventionBasedTestProxy.test(ConventionBasedTestProxyGenerator.java:120)
	at com.teradata.tempto.sample_table_insert.insert_0(Unknown Source)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- sql_tests.testcases.sample_table_insert.insert_0 -->
  <testcase name="sql_tests.testcases.sample_table_insert.insert_1" time="19.589" classname="com.teradata.tempto.sample_table_insert">
    <failure type="java.lang.RuntimeException" message="freemarker.core.InvalidReferenceException: The following has evaluated to null or missing:
==&amp;gt; mutableTables.sample_table_prepared  [in template &amp;quot;name&amp;quot; at line 1, column 17]

----
Tip: It&amp;apos;s the step after the last dot that caused this error, not those before it.
----
Tip: If the failing expression is known to be legally refer to something that&amp;apos;s null or missing, either specify a default value like myOptionalVar!myDefault, or use &amp;lt;#if myOptionalVar??&amp;gt;when-present&amp;lt;#else&amp;gt;when-missing&amp;lt;/#if&amp;gt;. (These only cover the last step of the expression; to cover the whole expression, use parenthesis: (myOptionalVar.foo)!myDefault, (myOptionalVar.foo)??
----

----
FTL stack trace (&amp;quot;~&amp;quot; means nesting-related):
	- Failed at: ${mutableTables.sample_table_prepared}  [in template &amp;quot;name&amp;quot; at line 1, column 15]
----">
      <![CDATA[java.lang.RuntimeException: freemarker.core.InvalidReferenceException: The following has evaluated to null or missing:
==> mutableTables.sample_table_prepared  [in template "name" at line 1, column 17]

----
Tip: It's the step after the last dot that caused this error, not those before it.
----
Tip: If the failing expression is known to be legally refer to something that's null or missing, either specify a default value like myOptionalVar!myDefault, or use <#if myOptionalVar??>when-present<#else>when-missing</#if>. (These only cover the last step of the expression; to cover the whole expression, use parenthesis: (myOptionalVar.foo)!myDefault, (myOptionalVar.foo)??
----

----
FTL stack trace ("~" means nesting-related):
	- Failed at: ${mutableTables.sample_table_prepared}  [in template "name" at line 1, column 15]
----
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.resolveTemplates(SqlQueryConventionBasedTest.java:139)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.runTestQuery(SqlQueryConventionBasedTest.java:115)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.test(SqlQueryConventionBasedTest.java:94)
	at com.teradata.tempto.internal.convention.ConventionBasedTestProxyGenerator$ConventionBasedTestProxy.test(ConventionBasedTestProxyGenerator.java:120)
	at com.teradata.tempto.sample_table_insert.insert_1(Unknown Source)
	at com.teradata.tempto.runner.TemptoRunner.run(TemptoRunner.java:81)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:59)
	at com.teradata.tempto.runner.TemptoRunner.runTempto(TemptoRunner.java:47)
	at com.teradata.tempto.examples.TemptoExamples.main(TemptoExamples.java:30)
Caused by: freemarker.core.InvalidReferenceException: The following has evaluated to null or missing:
==> mutableTables.sample_table_prepared  [in template "name" at line 1, column 17]

----
Tip: It's the step after the last dot that caused this error, not those before it.
----
Tip: If the failing expression is known to be legally refer to something that's null or missing, either specify a default value like myOptionalVar!myDefault, or use <#if myOptionalVar??>when-present<#else>when-missing</#if>. (These only cover the last step of the expression; to cover the whole expression, use parenthesis: (myOptionalVar.foo)!myDefault, (myOptionalVar.foo)??
----

----
FTL stack trace ("~" means nesting-related):
	- Failed at: ${mutableTables.sample_table_prepared}  [in template "name" at line 1, column 15]
----
	at freemarker.core.InvalidReferenceException.getInstance(InvalidReferenceException.java:116)
	at freemarker.core.EvalUtil.coerceModelToString(EvalUtil.java:346)
	at freemarker.core.Expression.evalAndCoerceToString(Expression.java:82)
	at freemarker.core.DollarVariable.accept(DollarVariable.java:40)
	at freemarker.core.Environment.visit(Environment.java:312)
	at freemarker.core.MixedContent.accept(MixedContent.java:62)
	at freemarker.core.Environment.visit(Environment.java:312)
	at freemarker.core.Environment.process(Environment.java:290)
	at freemarker.template.Template.process(Template.java:312)
	at com.teradata.tempto.internal.convention.sql.SqlQueryConventionBasedTest.resolveTemplates(SqlQueryConventionBasedTest.java:133)
	... 29 more
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- sql_tests.testcases.sample_table_insert.insert_1 -->
  <testcase name="testInjection" time="0.099" classname="com.teradata.tempto.examples.InjectionTest"/>
</testsuite> <!-- all -->
